Containerfile.bluechi-rpm-build

```diff
@@ -0,0 +1,37 @@
+FROM quay.io/centos/centos:stream9
```

Comment:
Did you try to work with https://quay.io/repository/bluechi/build-base ?
This way we would not need to maintain two `dnf install` statements.

Reply:
Awesome, thanks! Merging